--- svdrp.c.orig	2008-02-17 14:36:01.000000000 +0100
+++ svdrp.c	2008-05-02 16:29:16.000000000 +0200
@@ -1606,7 +1606,7 @@
         char buffer[BUFSIZ];
         gethostname(buffer, sizeof(buffer));
         time_t now = time(NULL);
-        Reply(220, "%s SVDRP VideoDiskRecorder %s; %s", buffer, VDRVERSION, *TimeToString(now));
+        Reply(220, "%s SVDRP VideoDiskRecorder %s; %s; %s", buffer, VDRVERSION, *TimeToString(now), I18nCharSets()[Setup.OSDLanguage]);
         }
      if (NewConnection)
         lastActivity = time(NULL);
